pkg/reconciler/inmemorychannel/controller/inmemorychannel.go

```diff
@@ -212,6 +213,19 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, imc *v1.InMemoryChannel)
 		imc.Status.Address = &httpAddress
 	}
 
+	if featureFlags.IsOIDCAuthentication() {
+		logging.FromContext(ctx).Error("OIDC enabled")
+		audience := auth.GetAudience(v1.SchemeGroupVersion.WithKind("InMemoryChannel"), imc.ObjectMeta)
+
+		logging.FromContext(ctx).Debugw("Setting the imc audience", zap.String("audience", audience))
+		imc.Status.Address.Audience = &audience
+	} else {
+		logging.FromContext(ctx).Error("OIDC disabled")
```

Comment:
Error level log? it seems a normal state

Reply:
we don't need this lines at all as we have some more informative logs right after

Reply:
updated it